### PR TITLE
add Trainer model TrainerDTO and tests

### PR DESCRIPTION
**Pull request by PES34**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -45,6 +45,7 @@
 
 	</dependencies>
 
+
```
Comment (adamkulawik) [inline pom.xml:48]: removed that line

File: src/main/java/com/inqoo/trainingservice/app/DTO/TrainerDTO.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
+package com.inqoo.trainingservice.app.DTO;
+
+import java.util.Objects;
+
+public class TrainerDTO {
+    public String firstName;
```
Comment (adamkulawik) [inline src/main/java/com/inqoo/trainingservice/app/DTO/TrainerDTO.java:6]: all fields should be private final

File: src/main/java/com/inqoo/trainingservice/app/DTO/TrainerDTO.java [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+        this.firstName = firstName;
+        this.lastName = lastName;
+        this.experience = experience;
+    }
+
+    public String getFirstName() {
+        return firstName;
+    }
+
+    public void setFirstName(String firstName) {
```
Comment (adamkulawik) [inline src/main/java/com/inqoo/trainingservice/app/DTO/TrainerDTO.java:20]: try to remove setters, maybe we can live without them

File: src/main/java/com/inqoo/trainingservice/app/models/Trainer.java [hunk 4]
```diff
@@ -0,0 +1,88 @@
+package com.inqoo.trainingservice.app.models;
+
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+public class Trainer {
+
```
Comment (adamkulawik) [inline src/main/java/com/inqoo/trainingservice/app/models/Trainer.java:9]: empty line

File: src/main/java/com/inqoo/trainingservice/app/service/TrainerService.java [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+import com.inqoo.trainingservice.app.exception.TooLongDescriptionException;
+import com.inqoo.trainingservice.app.models.Trainer;
+import com.inqoo.trainingservice.app.repository.TrainerRepository;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class TrainerService {
```
Comment (adamkulawik) [inline src/main/java/com/inqoo/trainingservice/app/service/TrainerService.java:13]: maybe doesn't need to be public

File: src/test/java/com/inqoo/trainingservice/app/service/TrainerServiceIT.java [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+                generatedTxt);
+
+        //then
+        Assertions.assertThrows(TooLongDescriptionException.class, () -> {
+            trainerService.saveNewTrainer(trainerConverter.dtoToEntity(trainer1));
+        });
+    }
+
+    @Test
+    public void shouldUpdateInformationAboutTrainer(){
```
Comment (adamkulawik) [inline src/test/java/com/inqoo/trainingservice/app/service/TrainerServiceIT.java:78]: Why empty?

File: src/main/java/com/inqoo/trainingservice/app/models/Trainer.java [hunk 7]
```diff
@@ -0,0 +1,85 @@
+package com.inqoo.trainingservice.app.models;
+
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+public class Trainer {
```
Comment (adamkulawik) [inline src/main/java/com/inqoo/trainingservice/app/models/Trainer.java:8]: can be non public

File: src/main/java/com/inqoo/trainingservice/app/converter/TrainerConverter.java [hunk 8]
```diff
@@ -0,0 +1,24 @@
+package com.inqoo.trainingservice.app.converter;
+
+import com.inqoo.trainingservice.app.DTO.TrainerDTO;
+import com.inqoo.trainingservice.app.models.Trainer;
+import org.springframework.stereotype.Component;
+
+@Component
+public class TrainerConverter {
```
Comment (adamkulawik) [inline src/main/java/com/inqoo/trainingservice/app/converter/TrainerConverter.java:8]: can be non public

File: src/test/java/com/inqoo/trainingservice/app/service/TrainerServiceIT.java [hunk 9]
```diff
@@ -0,0 +1,82 @@
...
+                "Kowalski",
+                generatedTxt);
+
+        //then
+        Assertions.assertThrows(TooLongDescriptionException.class, () -> {
+            trainerService.saveNewTrainer(trainerConverter.dtoToEntity(trainer1));
+        });
+    }
+
+//    @Test
```
Comment (adamkulawik) [inline src/test/java/com/inqoo/trainingservice/app/service/TrainerServiceIT.java:78]: just remove it please

File: src/main/java/com/inqoo/trainingservice/app/DTO/TrainerDTO.java [hunk 10]
```diff
@@ -0,0 +1,42 @@
+package com.inqoo.trainingservice.app.DTO;
+
+import java.util.Objects;
+
+public class TrainerDTO {
+    private final String firstName;
+    private final String lastName;
+    private final String experience;
+
+
```
Comment (adamkulawik) [inline src/main/java/com/inqoo/trainingservice/app/DTO/TrainerDTO.java:10]: empty lines

File: src/main/java/com/inqoo/trainingservice/app/converter/TrainerConverter.java (hunk 8 above)
Comment (PES34) [reply src/main/java/com/inqoo/trainingservice/app/converter/TrainerConverter.java:10]: must be public

File: src/main/java/com/inqoo/trainingservice/app/DTO/TrainerDTO.java (hunk 2 above)
Comment (PES34) [reply src/main/java/com/inqoo/trainingservice/app/DTO/TrainerDTO.java]: done

**Review by adamkulawik — APPROVED** [review]

**Pull request merged**